packages/cxl-ui/scss/cxl-dashboard-notification.scss

```
@@ -0,0 +1,74 @@
+@use "~@conversionxl/cxl-lumo-styles/scss/mq";
+
+:host {
+  .notification {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: var(--lumo-space-s);
+    margin: 0;
+    color: var(--lumo-primary-contrast-color);
+    background-color: var(--cxl-color-dark-gray);
+  }
+
+  .icon-bell {
+    position: relative;
+    margin-right: 14px;
+
+    &::after {
+      position: absolute;
+      top: 4px;
+      right: -7px;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      width: 12px;
+      height: 12px;
+      font-size: 8px;
+      font-weight: 700;
+      line-height: normal;
+      color: var(--lumo-tint);
+      content: attr(count);
+      background-color: var(--lumo-primary-color);
+      border: 2px solid var(--cxl-color-dark-gray);
+      border-radius: 100%;
+    }
+  }
+
+  vaadin-details[opened] {
+    .notification {
+      color: var(--lumo-primary-color);
+      background-color: var(--cxl-color-light-pink);
+      border-bottom-right-radius: 0;
+      border-bottom-left-radius: 0;
+    }
+
+    .icon-bell::after {
+      border-color: var(--cxl-color-light-pink);
+    }
+
+    .icon-arrow {
+      transform: rotate(90deg);
+    }
+  }
+
+  .tabs-wrapper {
+    max-height: 628px; //To fit ~6 cards in the container
```

Can we use `calc()` here with vars which define notifications items or it's too complex?

```suggestion
    max-height: 628px; // Fit ~6 cards in the container.
```

---

Mathias changed the Figma component. Now it will either be the necessary height to fit the contents or 40px from the bottom if it will be taller than screen height.